# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/pyrequests/setup.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- requests-2.28.0/setup.py.orig	2022-06-24 20:22:12.526802052 +0200
+++ requests-2.28.0/setup.py	2022-06-24 20:22:28.019801965 +0200
@@ -95,7 +95,6 @@
     package_dir={"requests": "requests"},
     include_package_data=True,
     python_requires=">=3.7, <4",
-    install_requires=requires,
     license=about["__license__"],
     zip_safe=False,
     classifiers=[
